iowa-a/bedrock-prod/canary-deploy.yaml

```diff
@@ -223,11 +223,11 @@ spec:
             port: 8000
         resources:
           limits:
-            cpu: 1250m
-            memory: 800Mi
+            cpu: 1500m
+            memory: 2500Mi
           requests:
-            cpu: 1000m
-            memory: 600Mi
+            cpu: 1400m
+            memory: 2000Mi
```

Comment:
How large are our instances? Will requiring this much CPU and memory make it harder to schedule the pods or binpack inefficiently?

Not a huge concern at canary stage.

Reply:
aws have ACPU of 3920/14751 per node.  

gcp is quite a bit smaller, 3920 cpu/6061 memory per node.  So this will likely be a little too large for efficent packing in gcp.  Hopefully we find the canaries show good performance then we can back down a bit on the worker count and resources before rolling to prod.